frontend/src/pages/Landing/index.js

```diff
@@ -84,12 +73,6 @@ function Landing() {
         topic: 'region',
         condition: 'is',
         query: defaultRegion,
-      },
-      {
-        id: uuidv4(),
-        topic: 'startDate',
-        condition: 'is within',
-        query: defaultDate,
       }]
       : getFiltersWithAllRegions(),
   );
```

Comment:
it doesn't really matter, but I think you could probably remove this ```getFiltersWithAllRegions()``` function and just put the ```allRegionsFilters``` array in the ternary here, since all the function does now is copy the all regions array